monkestation/code/datums/status_effects.dm

```diff
@@ -0,0 +1,30 @@
+
+/datum/status_effect/kleptomania
+	id = "kleptomania"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = null
+
+/datum/status_effect/kleptomania/tick()
+	if(!(owner.incapacitated()) && prob(10))
```

Comment:
A lot of these if statements could probably be combined.
Also, have the prob(10) first since that's the main check and will fail more often.